blagues.json

```
@@ -6038,7 +6038,421 @@
+    "answer": "... je suis malade.\""
+  },
+  {
+    "id": 1047,
+    "type": "dark",
+    "joke": "Pourquoi Jésus n'a pas joué le match Bethléem-Jérusalem ?",
+    "answer": "Car il était suspendu."
+  },
+  {
+    "id": 1048,
+    "type": "dark",
+    "joke": "Quelle est la différence entre un lycéen et un cancéreux ?",
+    "answer": "Il n'y en a pas, l'année prochaine ils passent tous les deux en phase terminale."
+  },
+  {
+    "id": 1049,
+    "type": "dark",
+    "joke": "Quel est le point commun entre un ami et un arbre ?",
+    "answer": "Ils tombent tous les deux au sol quand on abat à la hache !"
+  },
+  {
+    "id": 1050,
+    "type": "limit",
+    "joke": "Que dit un éléphant a un nudiste ?",
+    "answer": "\"Alors c'est avec ça que tu bois ?\""
+  },
+  {
+    "id": 1051,
+    "type": "dark",
+    "joke": "Pourquoi faut-il toujours poursuivre un arabe sur un scooter ?",
+    "answer": "Car c'est probablement le tien."
+  },
+  {
+    "id": 1052,
+    "type": "dark",
+    "joke": "Il fait arrêter les blagues sur les homosexuels.",
+    "answer": "Ils en ont plein le cul."
+  },
+  {
+    "id": 1053,
+    "type": "dark",
+    "joke": "Comment faire rentrer 20 juifs dans la voiture ?",
+    "answer": "Deux a l'avant, trois à l'arrière, deux dans le coffre et tous les autres dans le cendrier !"
+  },
+  {
+    "id": 1054,
+    "type": "global",
+    "joke": "Quel est le comble pour un breton ?",
+    "answer": "Perdre son parapluie."
+  },
+  {
+    "id": 1055,
+    "type": "global",
+    "joke": "Pourquoi les vaches ferment-t'elles les yeux ?",
+    "answer": "Pour faire du lait concentré."
+  },
+  {
+    "id": 1056,
+    "type": "limit",
+    "joke": "Quel est la différence entre un vieux et un renard ?",
+    "answer": "Le renard a la queue touffue et le vieux a la queue foutue."
+  },
+  {
+    "id": 1057,
+    "type": "global",
+    "joke": "J'avais une blague sur le confinement.",
+    "answer": "Elle n'est jamais sortie."
+  },
+  {
+    "id": 1058,
+    "type": "limit",
+    "joke": "Comment appelle t-on une actrice porno qui tombe enceinte ?",
+    "answer": "Un accident de travail."
+  },
+  {
+    "id": 1059,
+    "type": "limit",
+    "joke": "Pourquoi les femmes parlent-elle autant et les hommes réfléchissent-ils autant ?",
+    "answer": "Parce que les femmes ont quatre lèvres et les hommes deux cerveaux."
+  },
+  {
+    "id": 1060,
+    "type": "dark",
+    "joke": "La vie c'est comme une boite de chocolat !",
+    "answer": "Personne ne veut des noirs !"
+  },
+  {
+    "id": 1061,
+    "type": "beauf",
+    "joke": "Quelle est la différence entre des haricots et un pet ?",
+    "answer": "Environ 1H"
+  },
+  {
+    "id": 1062,
+    "type": "beauf",
+    "joke": "Pourquoi les tornades ont-elles des noms féminins ?",
+    "answer": "pour deux raisons, la première parce qu’elles arrivent chaudes et humides et la deuxième parce qu’elles repartent avec tes affaires."
+  },
```

```suggestion
  {
    "id": 1062,
    "type": "beauf",
    "joke": "Pourquoi les tornades ont-elles des noms féminins ?",
    "answer": "Pour deux raisons, la première parce qu’elles arrivent chaudes et humides et la deuxième parce qu’elles repartent avec tes affaires."
  },
```
Après, la réponse à la blague est peut-être un peu longue